### PR TITLE
grammar and style edits

### PR DESCRIPTION
**Pull request by stationaery**
grammar and style edits

### CODE REVIEW

File: call-for-submissions.md [hunk 1]
```diff
@@ -69,9 +67,7 @@ Here are some examples of content we typically reject:
 
 ### How to submit
 
-If you can share something with your fellow workers, we help you figure out a good story and strategy.
-
-Here’s how to start the process:
+<markHere’s how to start the process:</mark>
```
Comment (jessesquires) [inline call-for-submissions.md:70]: syntax error

```suggestion
<mark>Here’s how to start the process:</mark>
```

File: call-for-submissions.md [hunk 2]
```diff
@@ -81,6 +77,6 @@ Here’s how to start the process:
 1. Finally, we publish the piece! Our newsletters currently comes out every other Friday.
 
 
-### Other ways to contribute
+### More ways to contribute
 
-In addition to workers’ perspective, we welcome collaborators who can help with news, history, and songs. We also could use more eyes on proofreading and more hands pushing 'publish' each week! Engaging more collaborators is how we ensure the stewardship of this project for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
+In addition to workers’ perspective, we welcome collaborators who can help pull together news, histories, and songs – as well more eyes on proofreading and more hands pushing 'publish' each week! Including collaborators is how we grow our collective and steward this newsletter for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
```
Comment (jessesquires) [inline call-for-submissions.md:82]: "histories" sounds weird and i think it's confusing.

```suggestion
In addition to workers’ perspective, we welcome collaborators who can help pull together news, history, and songs &mdash; as well more eyes on proofreading and more hands pushing 'publish' each week! Including collaborators is how we grow our collective and steward this newsletter for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
```

Comment (stationaery) [reply call-for-submissions.md:82]: True. Let's leave it as history.

**Comment by stationaery** [comment]
Good catches, @jessesquires 👀

**Comment by jessesquires** [comment]
@stationaery did you mean to close this? or merge?

**Comment by stationaery** [comment]
I meant to merge. Wrong button, sorry, still getting the hang of this 😬

**Comment by anaulin** [comment]
Seems like @stationaery 's intent was to merge this, so I'll do that in a second. (I just looked at it locally, seems fine.)

Reading through this, I note that we are on this page using the term "workers' perspective" (with plural workers), but the newsletter itself has used "worker's perspective" (singular worker), and more recently it seems we've used one or the other interchangeably.

I don't have a strong feeling either way, but it would be good to standardize on something (I find the inconsistency distracting, others might too).

**Review by anaulin — APPROVED** [review]

**Comment by dellsystem** [comment]
@anaulin I think about the worker('s) perspective thing all the time!! good to know i'm not the only one :D

**Comment by stationaery** [comment]
LOL!!!1 Searching "workers perspective" the first to results are "Workers'
perspective." I do like the collective conjugation of the phrase but also
admit it's a bit wonky at first glance. The first newsletter volume used
both 's and 's and in fact we've used both, too. Anyone have strong
feelings about which conjugation best communicates the spirit of the
perspective that working people offer? (Perhaps we put it to a poll on
Twitter?)

**Comment by anaulin** [comment]
I think it makes sense to have both in plural "workers' perspectives" when
it is multiple perspectives from multiple people, and also when referring
in general to what we do (e.g. "this newsletter centers workers'
perspectives").

To me it feels more natural to call it "worker's perspective" in singular
when it is a piece by one worker. But that might be arguable?

"Workers' perspective" (plural workers, but singular perspective) feels
like the most troublesome variant to me. Like, more than one worker is
going to have more than one perspective, I think? But then if it is just
one piece in the newsletter, it also doesn't make sense to call it a plural
"perspectives", either. So, uhm, 🤔 🤷🏼‍♀️

**Comment by jessesquires** [comment]
lol. apostrophes

**Pull request merged**